pkg/controller/backup_deletion_controller.go

```diff
@@ -442,6 +457,66 @@ func (c *backupDeletionController) deleteResticSnapshots(backup *v1.Backup) []er
 	return errs
 }
 
+func getCSIVolumeSnapshotsInBackup(b *velerov1api.Backup, csiClient snapshotter.SnapshotV1beta1Interface, log *logrus.Entry) ([]snapshotv1beta1api.VolumeSnapshot, []error) {
+	csiVolSnaps := []snapshotv1beta1api.VolumeSnapshot{}
+	errs := []error{}
+
+	selector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			velerov1api.BackupNameLabel: label.GetValidName(b.Name),
+		},
+	}
+	listOpts := metav1.ListOptions{
+		LabelSelector: metav1.FormatLabelSelector(&selector),
+	}
+
+	for _, ns := range pkgbackup.GetNamespacesInBackup(b) {
+		log.Debugf("Listing CSI volumesnapshots in namespace %s", ns)
+		nsvs, err := csiClient.VolumeSnapshots(ns).List(listOpts)
+		if err != nil {
+			errs = append(errs, err)
```

Comment:
`errs = append(errs, errors.WithStack(err))` 

Using errors.WithStack, errors.Wrap, etc. will add a stack trace to the error. We generally wrap any errors that come back from third-party libraries, since they presumably won't have the stack info, but it's unnecessary to wrap errors returned from Velero-internal code since presumably they're already have stack info.